### ISSUE TITLE
[BUG]: java.lang.RuntimeException: Raytracing pipeline 0 has an unexpected descriptor set layout at set 0

### ISSUE THREAD

**Issue by Walkersifolia** [MCRcortex/vulkanite#31]
I run the game with mods fabric-api-0.89.0, indium-1.0.27, iris-1.6.9, sodium-0.5.3, vulkanite-0.0.4, and when I try to open the shaders, game shows me 'java.lang.RuntimeException: Raytracing pipeline 0 has an unexpected descriptor set layout at set 0'.

But when I use the indium-1.0.25, iris-1.6.8, sodium-0.5.2, vulkanite-0.0.3, I don't have this problem.

My PC specs is i5-12400F, 32GB DDR4 3600MHz, RTX 4060

**Comment by Walkersifolia** [comment]
If you need, I can upload my complete mod pack

**Comment by Aspirata** [comment]
I'm also getting this error

**Comment by Walkersifolia** [comment]
> I'm also getting this error


Yes, although 0.0.4 has not been officially released yet, solving it now can avoid this problem when it is released.

**Comment by BalintCsala** [comment]
@Walkersifolia @Aspirata this is not an issue if you use packs meant for the current version. Stay on 0.0.3 until the release when VDP will update simultaneously

**Comment by Walkersifolia** [comment]
> @Walkersifolia @Aspirata this is not an issue if you use packs meant for the current version. Stay on 0.0.3 until the release when VDP will update simultaneously

Yeah, I know, and I said I can run with 0.0.3, I just report the problem in develop.
